Converted CALLOC to MALLOC

pbhatia22 opened the pull request:
Converted calloc to malloc as we
already initialized all fields after allocation

Updates: #1000
Change-Id: I3e250fb016f6ea7e949983fcb8e98f1018dbad05
Signed-off-by: Preet Bhatia <pbhatia@redhat.com>

pbhatia22 committed:
Converted CALLOC to MALLOC

Converted calloc to malloc as we
already initialized all fields after allocation

Change-Id: I3e250fb016f6ea7e949983fcb8e98f1018dbad05
Updates: #1000
Signed-off-by: Preet Bhatia <pbhatia@redhat.com>

pbhatia22 commented:
/run regression

mykaul commented on libglusterfs/src/syncop-utils.c:
```diff
@@ -294,7 +294,7 @@ _run_dir_scan_task(call_frame_t *frame, xlator_t *subvol, loc_t *parent,
     int ret = 0;
     struct syncop_dir_scan_data *scan_data = NULL;
 
-    scan_data = GF_CALLOC(1, sizeof(struct syncop_dir_scan_data),
```
Looks OK, though I wish lines below 304-314 were setting variables in the order they are in the struct.

pbhatia22 replied:
Updated

mykaul reviewed (dismissed):
LGTM. Generally, we should ensure initialization happens in the order variables are in the struct.

pbhatia22 committed:
Converted CALLOC to MALLOC

Converted calloc to malloc as we
already initialized all fields after allocation

Updates: #1000
Change-Id: I3e250fb016f6ea7e949983fcb8e98f1018dbad05
Signed-off-by: Preet <pbhatia@redhat.com>

pbhatia22 commented:
/run regression

gluster-ant commented:
1 test(s) failed 
./tests/bugs/glusterd/brick-order-check-add-brick.t

0 test(s) generated core 


3 test(s) needed retry 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_nfs_bug-1116503.t
./tests/bugs/glusterd/brick-order-check-add-brick.t

2 flaky test(s) marked as success even though they failed 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_nfs_bug-1116503.t
https://build.gluster.org/job/gh_centos7-regression/2415/

pbhatia22 commented:
/run regression
